.travis.yml

```diff
@@ -4,7 +4,6 @@ python:
   - 3.3
   - 3.4
   - 3.5
-  - 3.6
```

Comment:
We already spotted that, but failed to use tox-travis with this information and a usable tox.ini. `nightly` would not match the `py36` TOXENV variable.